src/test/java/org/opensearch/flowframework/workflow/DeployModelStepTests.java

```diff
@@ -215,22 +200,7 @@ public void testDeployModelTaskFailure() throws IOException, InterruptedExceptio
         // Stub getTask for success case
         doAnswer(invocation -> {
             ActionListener<MLTask> actionListener = invocation.getArgument(1);
-            MLTask output = new MLTask(
-                taskId,
-                modelId,
-                null,
-                null,
-                MLTaskState.FAILED,
-                null,
-                null,
-                null,
-                null,
-                null,
-                null,
-                "error",
-                null,
-                false
-            );
+            MLTask output = MLTask.builder().taskId(taskId).modelId(modelId).state(MLTaskState.COMPLETED).async(false).build();
```

Comment:
looks like these are all subtly different parameters so you can't copy paste...

Reply:
changed it